easybuild/easyblocks/n/namd.py

```diff
@@ -17,7 +17,7 @@
 import os
 import re
 import shutil
-from easybuild.tools import LooseVersion
+from distutils.version import LooseVersion
```

Comment:
So, the important thing when trying to contribute upstream is that you should base your easyblocks on the latest from development. This currently reverts things that have been fixed, like this change,.

Reply:
I have made the changes in the latest commit.